src/fosslight_scanner/fosslight_scanner.py

```diff
@@ -255,60 +263,114 @@ def init(output_path=""):
     return os.path.isdir(_output_dir), output_root_dir, result_log
 
 
+def check_comapre_output_file(output_file_name, format):
+    CUSTOMIZED_FORMAT_FOR_COMPARE_MODE = {'excel': '.xlsx', 'html': '.html', 'json': '.json', 'yaml': '.yaml'}
+
+    if format == "":
+        logger.info(" * Default result format : excel")
+        format = "excel"
+
+    success, msg, output_path, output_file, output_extension = check_output_format(output_file_name, format,
+                                                                                   CUSTOMIZED_FORMAT_FOR_COMPARE_MODE)
+    success, final_excel_dir, result_log = init(output_path)
+    rmdir(_output_dir)
```

Comment:
(Minor) How about adding a parameter to `init `and adding an option that doesn't create `_output_dir`?
This is to avoid the need to create and delete unnecessary directories.

Reply:
I changed it.